src/pages/Booking/Booking.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import styled from 'styled-components';
+import Calendar from './components/Calendar/Calendar';
+import Movie from './components/Movie/Movie';
+
+// [컴포넌트 분리]
+// 1. 달력(월, 달력 아이콘)
+// 2. 영화
+// 3. 극장
+// 4. 시간
+
+function Booking() {
+  return (
+    <div>
+      <Wrapper primary bg>
+        <TopCategory>
+          <TopItems>
+            <Icon className="fas fa-home" primary></Icon>
+          </TopItems>
+          <TopItems>
+            <Icon className="fas fa-chevron-right"></Icon>
+            <span>예매</span>
+          </TopItems>
+          <TopItems>
+            <Icon className="fas fa-chevron-right"></Icon>
+            <span>빠른예매</span>
+          </TopItems>
+        </TopCategory>
+      </Wrapper>
+      <Wrapper primary>
+        <Title primary>빠른예매</Title>
+        <Calendar />
+        <Movie />
+      </Wrapper>
+    </div>
+  );
+}
+
+const Wrapper = styled.div`
+  padding: ${props => (props.primary ? '0  20%' : '0  0 ')};
+  background: ${props => props.bg && '#f8f8fa'};
```

Comment:
```suggestion
  padding: ${({ primary }) => (primary ? '0  20%' : '0  0 ')};
  background: ${({ bg }) => bg && '#f8f8fa'};
```
함수 파라미터 자리 바로 구조분해 할당 가능합니다